magento/magento2
# Added identification ID of the provided option type by its sku in the specified store.

Description:
1. Added optionTypeSkus private property to the class app/code/Magento/CatalogImportExport/Model/Import/Product/Option.php

2. Added $optionSku parameter to the getExistingOptionTypeId function.

3. Added identification ID of the provided option type by its sku in the specified store.

Fixed Issues
1. magento/magento2#23601: Magento2 Impossible to update custom option value based on custom option SKU

Manual testing scenarios:
1.Create product with Customizable Options
2. Check option_type_id in table catalog_product_option_type_value
3. Export this product
4. Edit in csv file custom options title 
5. Save scv file and import it
6. Check option_type_id in table catalog_product_option_type_value

Expected result:
Value option_type_id is not changed when row is updated
